pkg/network/setup/netpod/netpod.go

```diff
@@ -279,6 +279,8 @@ func (n NetPod) composeDesiredSpec(currentStatus *nmstate.Status) (*nmstate.Spec
 		case iface.DeprecatedPasst != nil:
 			spec.LinuxStack.IPv4.PingGroupRange = []int{107, 107}
 			spec.LinuxStack.IPv4.UnprivilegedPortStart = pointer.P(0)
+		// Passt is removed in v1.3. This scenario is tracking old VMIs that are still processed in the reconcile loop.
+		case iface.DeprecatedPasst != nil:
```

Comment:
You added a new case instead of commenting the existing one.

Reply:
My bad, fixed

Reply:
Fixed